conf/postgresql.application.conf

```diff
@@ -94,6 +94,7 @@ hostRepository.type = "com.arpnetworking.metrics.portal.hosts.impl.DatabaseHostR
 # Alerts
 # ~~~~~
 alertRepository.type = "com.arpnetworking.metrics.portal.alerts.impl.DatabaseAlertRepository"
+alertExecutionRepository.type = "com.arpnetworking.metrics.portal.alerts.impl.DatabaseAlertExecutionRepository"
```

Comment:
This just reminded me that I need to add `NoAlertExecutionRepository` as a default.